www/plas02/styless_resume.css

```diff
@@ -0,0 +1,213 @@
+h1 {
+    font-family: 'Times New Roman', Times, serif;
+    color: rgb(71, 6, 97);
+    font-size: 50px;
+    text-align: center;
+}
+
+h2 {
+
+    font-family: 'Times New Roman', Times, serif;
+    font-size: 15px;
+    text-align: center;
+    
+}
+
+
+.navigation {
+    list-style-type: none;
+    overflow: hidden;
+    background-color: rgba(240, 248, 255, 0.568);
+}
+
+li {
+    list-style: none;
+
+}
+
+li a {
+    float: left;
+    display: block;
+    color: rgb(29, 198, 204);
+    text-align: center;
+    padding: 8px 200px;
+    font-size: 24px;
+    text-decoration: none;
+    margin-bottom: 8px;
+}
+
+li a:hover {
+    color: rgb(59, 5, 109);
+}
+
+#cv_title {
+    font-size: 30px;
+    color:  rgb(71, 6, 97);
+    text-align: unset;
+}
+
+body {
+    background-image: url(https://cdn.pixabay.com/photo/2017/08/27/23/11/background-2687925_960_720.jpg);
```

Comment:
tenhle link na obrazek nekdy nefunguje, asi jedine ho stahnout k sobe, protoze nemuzes si byt jista, ze ten obrazek tam navzdy bude dostupny (nemas nad tim kontrolou), stejne tak pro vsehcny externi odkazy

Reply:
Je mi jasné, že v případě propacovanhove webu, bych měla mít obrázky u sebe. Když bych to však posílala na github s obrázky uloženými u sebe, tak by se to nezobrazilo, že?

Reply:
Kdyz to posles tak to posles i s obrazky, a tim padem se zobrazi spravne